image_proc/launch/image_proc.launch.py

```diff
@@ -41,16 +41,24 @@
 
 
 def generate_launch_description():
+
+    arg_namespace = DeclareLaunchArgument(
+        name='namespace', default_value='',
+        description=('namespace for all components loaded')
+    )
+    
```

Comment:
linter seems to think that you have some spaces on this line (it shows in the individual second commit as well -although I can't actually see it highlighted here as part of the PR - which is odd)

Reply:
Yea I noticed the same thing when the `flake` test failed, just assumed it was a CI glitch. I've tried to retrigger by removing any sort of white space that may have come up